plugins/CircuitExplorer/components/MorphologyCircuitComponent.cpp

```diff
@@ -101,6 +101,33 @@ void MorphologyCircuitComponent::onDestroy()
     }
 }
 
+void MorphologyCircuitComponent::onInspect(
+    const brayns::Vector3f &hit,
+    OSPGeometricModel modelHandle,
+    uint32_t primitiveID,
+    brayns::JsonObject &writeResult) const noexcept
+{
+    (void)hit;
+    (void)primitiveID;
+
+    auto morphBegin = _morphologies.begin();
+    auto morphEnd = _morphologies.end();
+    auto it = std::find_if(
+        morphBegin,
+        morphEnd,
+        [&](const MorphologyGeometry &morphology) { return morphology.model == modelHandle; });
+
+    if (it == morphEnd)
+    {
+        return;
+    }
+
+    auto index = std::distance(morphBegin, it);
+
+    auto cellId = _ids[index];
+    writeResult.set("neuron id", brayns::JsonValue(cellId));
```

Comment:
Same here, cast is not necessary if cellId is an integer type.

Why do you call it "neuron id" and not "neuron_id" is it not supposed to be a JSON-RPC property ?